library/create_kafka.py

```diff
@@ -0,0 +1,239 @@
+  - name: Create kafka
+    create_kafka:
+      name: "struttin"
+    register:
+      kafka_req_resp 
+
+ 
+'''
+
+RETURN = r'''
+# These are examples of possible return values, and in general should use other names for return values.
+original_message:
+    description: The original kafka request payload that was passed in.
+    type: dict
+    returned: always
+    sample: "invocation": {
+        "module_args": {
+            "billing_cloud_account_id": "",
+            "billing_model": "marketplace",
+            "cloud_provider": "aws",
+            "instance_type": "developer",
+            "marketplace": "",
+            "name": "struttin",
+            "plan": "",
+            "reauthentication_enabled": "True",
+            "region": "eu-west-1"
+        }
+message:
+    description: The output error / exception message that is returned in the case the module generates an error / exception.
+    type: dict
+    returned: in case of error / exception
+kafka_req_resp:
+    description: The response object from the Kafka_mgmt API.
+    "changed": true,
+        "failed": false,
+        "kafka_req_resp": {
+            "admin_api_server_url": "http://localhost:8000/data/kafka",
+            "billing_cloud_account_id": "",
+            "billing_model": "marketplace",
+            "bootstrap_server_host": "localhost:8000",
+            "browser_url": "http://localhost:8080/calbu9ccff6bdd4jsg30/dashboard",
+            "cloud_provider": "aws",
+            "created_at": "2020-10-05T12:51:24.053142+00:00",
+            "egress_throughput_per_sec": "1Mi",
+            "expires_at": "2022-06-18T05:27:01.816619+00:00",
+            "href": "/api/kafkas_mgmt/v1/kafkas/68Pr91QnNwW6Wis7J7jxM",
+            "id": "68Pr91QnNwW6Wis7J7jxM",
+            "ingress_throughput_per_sec": "1Mi",
+            "instance_type": "developer",
+            "instance_type_name": "Trial",
+            "kafka_storage_size": "10Gi",
+            "kind": "Kafka",
+            "marketplace": "",
+            "max_connection_attempts_per_sec": 50.0,
+            "max_data_retention_period": "P14D",
+            "max_partitions": 100.0,
+            "multi_az": false,
+            "name": "struttin",
+            "owner": "dsaridak",
+            "plan": "",
+            "reauthentication_enabled": true,
+            "region": "us-east-1",
+            "size_id": "x1",
+            "status": "ready",
+            "total_max_connections": 100.0,
+            "updated_at": "2020-10-05T12:56:36.362208+00:00",
+            "version": "3.0.1"
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+import rhoas_kafka_mgmt_sdk
+from rhoas_kafka_mgmt_sdk.api import default_api
+from rhoas_kafka_mgmt_sdk.model.kafka_request_payload import KafkaRequestPayload
+import auth.rhoas_auth as auth
+
+def run_module():
+    # define available arguments/parameters a user can pass to the module
+    module_args = dict(
+        name=dict(type='str', required=True),
```

Comment:
https://pypi.org/project/ansible-autodoc/

Reply:
looking into the doc gen 👍